### DISCUSSION TITLE
현재 게시물의 주인이랑 현재 유저랑 같은지 확인하는 api 필요

### DISCUSSION THREAD

**Discussion by hwangstar156** [woowacourse-teams/2022-gong-seek#59] [General]
필요해용

**Comment by jurlring** [comment]
이거 백엔드가 줘야하는건가용

**Comment by brorae** [comment]
프론트에서 액세스 토큰을 헤더에 담아서 보내면, 토큰에 있는 payload 값으로 유저를 식별하고, 그 유저와 게시물 작성자와 비교할 수 있을 것 같은데, 따로 필요한 이유가 있을까요?
